src/NumericalAlgorithms/DiscontinuousGalerkin/NumericalFluxes/NumericalFluxHelpers.hpp

```diff
@@ -0,0 +1,122 @@
+  numerical_flux_computer.package_data(
+      make_not_null(&get<PackageFieldTags>(packaged_data->field_data))...,
+      make_not_null(&get<PackageExtraTags>(packaged_data->extra_data))...,
+      args...);
+}
+
+template <typename NumericalFluxType, typename... AllFieldTags,
+          typename... AllExtraTags, typename... NormalDotNumericalFluxTypes,
+          typename... PackageFieldTags, typename... PackageExtraTags>
+void normal_dot_numerical_fluxes_impl(
+    const NumericalFluxType& numerical_flux_computer,
+    const dg::SimpleBoundaryData<tmpl::list<AllFieldTags...>,
+                                 tmpl::list<AllExtraTags...>>&
+        packaged_data_int,
+    const dg::SimpleBoundaryData<tmpl::list<AllFieldTags...>,
+                                 tmpl::list<AllExtraTags...>>&
+        packaged_data_ext,
+    tmpl::list<PackageFieldTags...> /*meta*/,
+    tmpl::list<PackageExtraTags...> /*meta*/,
+    const gsl::not_null<
+        NormalDotNumericalFluxTypes*>... n_dot_num_fluxes) noexcept {
+  numerical_flux_computer(
+      n_dot_num_fluxes...,
+      get<PackageFieldTags>(packaged_data_int.field_data)...,
+      get<PackageExtraTags>(packaged_data_int.extra_data)...,
+      get<PackageFieldTags>(packaged_data_ext.field_data)...,
+      get<PackageExtraTags>(packaged_data_ext.extra_data)...);
+}
+}  // namespace detail
+
+// @{
+/// Helper function to unpack arguments when invoking the numerical flux
+template <typename NumericalFluxType, typename... AllFieldTags,
+          typename... AllExtraTags, typename... Args>
+void package_data(
+    const gsl::not_null<dg::SimpleBoundaryData<tmpl::list<AllFieldTags...>,
+                                               tmpl::list<AllExtraTags...>>*>
+        packaged_data,
+    const NumericalFluxType& numerical_flux_computer,
+    const Args&... args) noexcept {
+  static_assert(
+      tt::conforms_to_v<NumericalFluxType, protocols::NumericalFlux>,
+      "The 'NumericalFluxType' must conform to 'dg::protocol::NumericalFlux'.");
+  detail::package_data_impl(packaged_data, numerical_flux_computer,
+                            typename NumericalFluxType::package_field_tags{},
+                            typename NumericalFluxType::package_extra_tags{},
+                            args...);
+}
+
+template <typename NumericalFluxType, typename... AllFieldTags,
+          typename... AllExtraTags, typename... NormalDotNumericalFluxTypes>
+void normal_dot_numerical_fluxes(
+    const NumericalFluxType& numerical_flux_computer,
+    const dg::SimpleBoundaryData<tmpl::list<AllFieldTags...>,
+                                 tmpl::list<AllExtraTags...>>&
+        packaged_data_int,
+    const dg::SimpleBoundaryData<tmpl::list<AllFieldTags...>,
+                                 tmpl::list<AllExtraTags...>>&
+        packaged_data_ext,
+    const gsl::not_null<
+        NormalDotNumericalFluxTypes*>... n_dot_num_fluxes) noexcept {
```

Comment:
Output arguments go first.

Reply:
removed this function for now since it's not necessarily needed. I was planning to use it in the future but will deal it then.